cmd/reconciler/ip.go

```diff
@@ -13,11 +13,11 @@ const defaultReconcilerTimeout = 30
 
 func main() {
 	kubeConfigFile := flag.String("kubeconfig", "", "the path to the Kubernetes configuration file")
-	logLevel := flag.String("log-level", "error", "the logging level for the `ip-reconciler` app. Valid values are: \"debug\", \"verbose\", \"error\", and \"panic\".")
+	logFile := flag.String("log-file", "/host/var/log/ip-reconciler.log", "File on host for ip-reconciler to log to")
```

Comment:
I don't understand why you are removing the log level parameter. It still makes sense.

Also, you cannot use this default - i.e. what if the `cronjob` spec does not feature the volume mount ? It will fail.

You should use a default on the container file system, but override that in the daemonset spec shipped in whereabouts.